README.md

```diff
@@ -3,16 +3,16 @@
 [![Build Status](https://travis-ci.com/ankitjain28may/plexus.svg?token=PPiqU7v6mcqyZ9sMHrpV&branch=master)](https://travis-ci.com/ankitjain28may/plexus)
```

Comment:
Change the badge of Travis to this current markdown badge.
[![Build Status](https://travis-ci.org/ncs-jss/plexus.svg?branch=master)](https://travis-ci.org/ncs-jss/plexus)
`[![Build Status](https://travis-ci.org/ncs-jss/plexus.svg?branch=master)](https://travis-ci.org/ncs-jss/plexus)`